clappr/clappr-ios
# Set the Loading plugin to be a Core Plugin

### Goal
Set the LoadingContainerPlugin to be a plugin of the Core.

### How to test
1. Run all unit and ui tests.
1. Run the app and check if the Spinner appears when the video is buffering and hides when is playing.
